config/helm/aws-node-termination-handler/README.md

```diff
@@ -84,6 +84,8 @@ Parameter | Description | Default
 `serviceAccount.annotations` | Specifies the annotations for ServiceAccount       | `{}`
 `procUptimeFile` | (Used for Testing) Specify the uptime file | `/proc/uptime`
 `securityContext.runAsUserID` | User ID to run the container | `1000`
-`securityContext.runAsGroupID` | Group ID to run the container | `1000` 
+`securityContext.runAsGroupID` | Group ID to run the container | `1000`
 `nodeSelectorTermsOs` | Operating System Node Selector Key | `beta.kubernetes.io/os`
 `nodeSelectorTermsArch` | CPU Architecture Node Selector Key | `beta.kubernetes.io/arch`
+`enablePrometheusServer` | If true, start an http server exposing `/metrics` endpoint for prometheus. | `false`
+`prometheusPortServer` | Replaces the default HTTP port for exposing prometheus metrics. | `9092`
```

Comment:
Also, since NTH is a daemonset with `host_networking=true`, Nothing else will be able to bind to :9092 in the root network namespace and since it's listening on all interfaces, users will need to have a firewall/security group configured on the nodes to block access to the /metrics endpoint. Users should be doing that anyways, but maybe worth putting a note in the docs.

Reply:
yeah, I can write something regarding that